src/where.c

```diff
@@ -2599,18 +2599,23 @@ static int whereLoopInsert(WhereLoopBuilder *pBuilder, WhereLoop *pTemplate){
   */
   if( pBuilder->pOrSet!=0 ){
     if( pTemplate->nLTerm ){
+#ifndef _FREEBSD_KERNEL
 #if WHERETRACE_ENABLED
```

Comment:
This seems like another #define to adjust in the include file.

Reply:
Removed the guard -- the -Wno-undef flag which was recently added in the Makefile prevents the #define to need to be included in the include file. If the #define existed in the include file, then the code inside several #ifdef WHERETRACE_ENABLED branches would be executed, which is incompatible with the kernel.